test/util/parser_test.go

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (C) 2017 XLAB, Ltd.
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+package util_test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/mikelangelo-project/capstan/util"
+	. "gopkg.in/check.v1"
+)
+
+func Test(t *testing.T) { TestingT(t) }
+
+type testingParserSuite struct{}
+
+var _ = Suite(&testingParserSuite{})
+
+func (s *testingParserSuite) TestParseEnvironmentList(c *C) {
+	m := []struct {
+		comment     string
+		envList     []string
+		expectedRes []string
```

Comment:
Use `map[string]string` to directly provide the target map of extracted env variables.

https://play.golang.org/p/s3JySD_4q0